operations/enterprises.md

```diff
@@ -1,68 +1,72 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
 # Enterprises
 
 ## Get all enterprises
 
-Returns all enterprises within scope of the `Access Token`, optionally filtered by enterprise identifiers and external identifiers. This operation uses [Pagination](../guidelines/pagination.md) and supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+Returns all enterprises within scope of the `Access Token`, optionally filtered by enterprise identifiers and external identifiers. This operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
```

Comment:
Are we dropping the reference to pagination now?

Reply:
Added back, we're missing in on few endpoints 😅